scalar.proto

```diff
@@ -0,0 +1,249 @@
+syntax = "proto3";
```

Comment:
This file is copied from the "scalar" repository

Reply:
How do you manage multiple versions of the proto file?
`scalar.proto` might keep changing, so I think managing the latest version would fail.

Reply:
@feeblefakie 

Thank you for your question.

I will keep the latest `scalar.proto` (and its `scalar.proto.ts` and generated files for the Node and browser runtime) in the main branch because we have to generate `scalar_grpc_pb.js` and `scalar_pb.js` anyway when `scalar.proto` is updated.

And, when a minor version, for example, 3.5.x, is released, a `3.5` branch will be created, and the `scalar.proto` (and its related files) for that version will be maintained in that branch.

Reply:
So, copying the proto file from `scalar` repo for each minor version?

Reply:
@feeblefakie 

Thank you.

I am thinking to release Node/Web SDKs from this repository for versions after 3.5 (included).

If this is acceptable to you, we don't need to port the versions before 3.5 (no need to copy all scalar.proto files from minor branches 3.0 ~ 3.4 of the scalar repository)

The scalar.proto will be synchronized with the "scalar" repository's correspondent according to the branches.